csharp/extractor/Semmle.Extraction.CIL/Entities/PdbSourceFile.cs

```diff
@@ -25,7 +25,7 @@ public override IEnumerable<IExtractionProduct> Contents
                 else
                     Context.TrapWriter.Archive(TransformedPath, text);
 
-                yield return Tuples.file_extraction_mode(this, 2);
+                yield return Tuples.file_extraction_mode(this, Context.Extractor.Mode | ExtractorMode.Pdb);
```

Comment:
Fantastic that we get rid of the magic number and rely on the flag, but shouldn't this only be ExtractorMode.Pdb?

Reply:
No; extractor mode is now a bit field, which allows for more information. For example, a PDB file could be extracted in either a QL test or not, and now we record all combinations.